lib/active_graphql/model/relation_proxy.rb

```diff
@@ -169,26 +169,37 @@ def to_graphql
         formatted_action(raw.meta(paginated: true)).to_graphql
       end
 
-      def order(order_params)
-        if order_params.is_a?(Hash)
-          order_by = order_params.keys.first
-          direction = order_params.values.first
-        else
-          order_by = order_params
-          direction = :asc
+      def order(*order_params)
+        order_params_attributes = order_params.flat_map do |order_param|
+          ordering_attributes(order_param)
         end
+        order_params_attributes.compact!
 
-        return chain(order_attributes: {}) if order_by.to_s.empty?
+        return chain(order_attributes: {}) if order_params_attributes.to_s.empty?
```

Comment:
since we are ordering by multiple fields, we should use `[]` as default value. Also, can you rename `order_attributes` in to `order_values` or `order_list` - name is a little bit misleading now
```suggestion
        return chain(order_values: []) if order_params_attributes.to_s.empty?
```